flink-clients/src/main/java/org/apache/flink/client/program/rest/RestClusterClient.java

```diff
@@ -149,29 +153,16 @@ public RestClusterClient(Configuration config, T clusterId) throws Exception {
 			config,
 			null,
 			clusterId,
-			new ExponentialWaitStrategy(10L, 2000L),
-			null);
-	}
-
-	public RestClusterClient(
-			Configuration config,
-			T clusterId,
-			LeaderRetrievalService webMonitorRetrievalService) throws Exception {
-		this(
-			config,
-			null,
-			clusterId,
-			new ExponentialWaitStrategy(10L, 2000L),
-			webMonitorRetrievalService);
+			new ExponentialWaitStrategy(10L, 2000L));
 	}
 
 	@VisibleForTesting
 	RestClusterClient(
 			Configuration configuration,
 			@Nullable RestClient restClient,
 			T clusterId,
-			WaitStrategy waitStrategy,
-			@Nullable LeaderRetrievalService webMonitorRetrievalService) throws Exception {
+			WaitStrategy waitStrategy
+	) throws Exception {
```

Comment:
I think this formatting style differs from the rest of the code in this file.

Reply:
Re-formatted